resume.html

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+	<title>
+		Left Shark | I do what I want
+	</title>
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+	<link href='https://fonts.googleapis.com/css?family=Delius+Unicase' rel='stylesheet' type='text/css'>
+	<link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
+</head>
+<body>
+
+
+	<img src="images/profile.svg">
+
+	<h1>Left Shark</h1>
+
+	<h2>I do what I want</h2>
+
+	<nav>
+		<a href="index.html">go left</a>
+		<a href="resume.html">other left</a>
+		<!--  <a href="credits.html">credits</a> -->
+	</nav>
+
+		<section>
```

Comment:
:hammer: As mentioned in the other html file, make sure the indentation is proper. The `<section>` tags should be brought back a but to match up with its sibling, `<nav>`.